packages/quantic/force-app/main/default/lwc/quanticDateFacetValue/quanticDateFacetValue.html

```diff
@@ -1,14 +1,21 @@
 <template>
-  <li class="slds-var-m-around_small slds-grid">
-    <div class="slds-col_bump-right">
-      <template if:true={isChecked}>
-        <input type="checkbox" onchange={onSelect} checked/>
-      </template>
-      <template if:false={isChecked}>
-        <input type="checkbox" onchange={onSelect}/>
-      </template>
-      <span class="slds-var-m-horizontal_medium nowrap">{start} - {end}</span>
-    </div>
-    <div class="slds-var-m-horizontal_small">({item.numberOfResults})</div>
-  </li>
+  <template if:true={isChecked}>
+    <li class="slds-var-m-vertical_small slds-grid">
+      <div class="slds-col">
+        <input type="checkbox" class="facet__checkbox" checked onchange={onSelect} />
```

Comment:
Why not a `<lightning-input type="checkbox" label="Basic option" name="input1"></lightning-input>`?